src/stream/src/executor/actor.rs

```diff
@@ -145,6 +149,7 @@ where
     pub fn new(
         consumer: C,
         subtasks: Vec<SubtaskHandle>,
+        stream_actor: PbStreamActor,
```

Comment:
Do we need to store the entire actor's proto definition?

Reply:
I've changed to only store the `mview_definition.`